packages/Search/src/details/DTK_DetailsTreeTraversal.hpp

```diff
@@ -60,7 +58,7 @@ struct TreeTraversal
      * Return the index of the leaf node.
      */
     KOKKOS_INLINE_FUNCTION
-    static int getIndex( BoundingVolumeHierarchy<DeviceType> bvh,
+    static int getIndex( BoundingVolumeHierarchy<DeviceType> const &bvh,
```

Comment:
That's right `bvh` passed here is already on the device.  Good catch!

Reply:
nice